inference/core/active_learning/README.md

```diff
@@ -0,0 +1,269 @@
+# Active Learning
+
+This feature is designed to facilitate the systematic collection of data from actual production distributions with 
```

Comment:
This description is quite abstract ("actual production distributions"). You can use more direct language to explain this feature.

> Inference lets you collect data automatically from production use cases for use in training new versions of your model. This is called active learning.

> By collecting data actively that you can later label and use in training, you can gather data representative of the environment in which your model is deployed. This information may help you boost model performance.